stl/inc/yvals_core.h

```diff
@@ -1081,8 +1081,11 @@
 #define __cpp_lib_atomic_value_initialization 201911L
 
 #if _HAS_CXX20
-#define __cpp_lib_atomic_float 201711L
-#define __cpp_lib_bind_front   201907L
+#define __cpp_lib_atomic_flag_test              201907L
+#define __cpp_lib_atomic_float                  201711L
+#define __cpp_lib_atomic_lock_free_type_aliases 201907L
+#define __cpp_lib_atomic_wait                   201907L
+#define __cpp_lib_bind_front                    201907L

```

Comment:
The new feature-test macros need to be tested in `tests/std/tests/VSO_0157762_feature_test_macros/test.cpp`.